apps/emqx_auth/src/emqx_authz/emqx_authz.erl

```diff
@@ -499,33 +499,57 @@ do_authorize(
     Module = authz_module(Type),
     emqx_metrics_worker:inc(authz_metrics, Type, total),
     try Module:authorize(Client, PubSub, Topic, Connector) of
+        %% for example emqx_authz_client_info without acl rule.
+        %% skip client tracing when client_info returns "skip" in non-superuser mode.
+        %% this trace log cause user confusion.
+        skip ->
+            do_authorize(Client, PubSub, Topic, Tail);
```

Comment:
```
2024-05-29T06:56:50.480307+00:00 [AUTHZ] 01234@172.100.239.1:53170 msg: authorization_module_nomatch, action: SUBSCRIBE(Q0), module: emqx_authz_client_info, topic:xxx/yyy, username: emqx_u1
2024-05-29T06:56:50.481417+00:00 [info] tag: AUTHZ, clientid: 01234, msg: authorization_permission_allowed, peername: 172.100.239.1:53170, username: emqx_u1, topic: xxx/yyy, source: file, action: SUBSCRIBE(Q0)
```
This log can cause misunderstanding, so we do not print logs when there is no acl in clientinfo

Reply:
Should we make `emqx_authz_client_info` return `ignore` instead of expecting `skip` here, so the trace a few lines below will at least log that it was evaluated?